src/components/Todos/Todos.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { TodoItem } from '../../TodoItem';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  onRemoveTodo: (todoId: number) => void
+  onCheckedTodo: (todoId: number) => void
+  tempTodoId: number | null
+  handleImputTodo: (e: React.ChangeEvent<HTMLInputElement>) => void
+};
+
+export const Todos: React.FC<Props> = ({
+  todos,
+  onRemoveTodo,
+  onCheckedTodo,
+  tempTodoId,
+  handleImputTodo,
+}) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings